lib/resorts/canada-olympic-park/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  selector: '.ancillary-flyout__status table:first-child tr:not(:first-child)',
+  parse: {
```

Comment:
I am pretty sure that `{ name: 0, status: 1 }` is liftie's default - you can probably just remove the `parse` element here

Reply:
Cool! Removed now